lib/config/virtual_container_host.go

```diff
@@ -155,6 +155,8 @@ type Connection struct {
 	ExtensionName string `vic:"0.1" scope:"read-only" key:"extension_name"`
 	// Whether the session connection is secure
 	Insecure bool `vic:"0.1" scope:"read-only" key:"insecure"`
+	// TargetThumbprint is the SHA-1 digest of the Target's public certificate
```

Comment:
Why SHA-1? It is no longer considered secure. We should go at least with SHA-256.